src/components/organisms/UiList/UiList.vue

```diff
@@ -3,45 +3,81 @@
     :is="tag"
     class="ui-list"
   >
-    <!-- @slot Use this slot to place list items. -->
-    <slot>
-      <template
-        v-for="(item, key) in itemsToRender"
-        :key="key"
+    <template v-if="hasLoader">
```

Comment:
@kubaszajna @sonya0504 Why we want to add a loader to UiList? We can it to #default slot in usage place. 
```
<UiList>
 <UiLoader v-if/>
</UiList>
```